footprints/templates/base.html

```diff
@@ -55,6 +55,16 @@
         <script src="https://search.sites.columbia.edu/cu-privacy-notice/cu-privacy-notice.js"></script>
     </head>
 
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-KP827X523M"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        gtag('config', 'G-KP827X523M');
+    </script>
+
```

Comment:
See comment on the ahemap pr

Reply:
These script tags need to be inside either the `<head>` or the `<body>` element.